nerfstudio/process_data/process_data_utils.py

```diff
@@ -62,12 +62,12 @@ def list_images(data: Path) -> List[Path]:
     """Lists all supported images in a directory
 
     Args:
-        data: Path to the directory of images.
+        data: Path to the directory of images. Nested folders are searched as well.
```

Comment:
I don't think we should search nested folders by default. Can you make this by default turned off, and make a new parameter `search_nested_folders: bool = False`?

Reply:
Sure, can do.